plugin/elasticity/cable.cc

```diff
@@ -26,6 +26,29 @@
 namespace mujoco::plugin::elasticity {
 namespace {
 
+
+//create color scale palette 
+void scalar2rgba(float rgba[4], mjtNum Scalar, mjtNum min, mjtNum max) {
```

Comment:
As mentioned in the main discussion, it would be great to remove the coloring part from this branch.